src/main/java/teammates/ui/webapi/UpdateFeedbackSessionAction.java

```diff
@@ -42,12 +44,33 @@ void checkSpecificAccessControl() throws UnauthorizedAccessException {
     public JsonResult execute() throws InvalidHttpRequestBodyException {
         String courseId = getNonNullRequestParamValue(Const.ParamsNames.COURSE_ID);
         String feedbackSessionName = getNonNullRequestParamValue(Const.ParamsNames.FEEDBACK_SESSION_NAME);
+        // TODO: Use the value here.
+        getBooleanRequestParamValue(Const.ParamsNames.NOTIFY_ABOUT_DEADLINES);
 
         FeedbackSessionAttributes feedbackSession = getNonNullFeedbackSession(feedbackSessionName, courseId);
 
         FeedbackSessionUpdateRequest updateRequest =
                 getAndValidateRequestBody(FeedbackSessionUpdateRequest.class);
 
+        Map<String, Instant> oldStudentDeadlines = feedbackSession.getStudentDeadlines();
+        Map<String, Instant> oldInstructorDeadlines = feedbackSession.getInstructorDeadlines();
+        Map<String, Instant> studentDeadlines = updateRequest.getStudentDeadlines();
+        Map<String, Instant> instructorDeadlines = updateRequest.getInstructorDeadlines();
+        try {
+            boolean hasExtraStudents = !oldStudentDeadlines.keySet()
+                    .containsAll(studentDeadlines.keySet());
+            boolean hasExtraInstructors = !oldInstructorDeadlines.keySet()
+                    .containsAll(instructorDeadlines.keySet());
```

Comment:
Does not need to be in `try` block?
Also, would a comment on the extra user check be needed?

Reply:
No it does not. But it also does not need to be outside. I was not completely sure where to put it. I just put it inside because I felt maybe it would be best to narrow the scope of the variables since it was possible. I can just put it outside though if you want.

>Also, would a comment on the extra user check be needed?

I can put it. Seems sensible to me.

Reply:
> I was not completely sure where to put it.

After reading a few times, I feel like no harm leaving it inside👍
Just add the comment so that the extra checks are never removed?

Reply:
Ok sure!